CSS/styles.css

```diff
@@ -394,3 +435,288 @@ textarea {
 ::placeholder {
   color: #b3bac5;
 }
+
+/* Desktop Size */
+
+@media only screen and (min-width: 1261px) {
+  :root {
+    --Neturals-1: #091e42;
+  }
+
+  body {
+    display: initial;
+  }
+
+  .hero-section,
+  #header-container {
+    width: 100%;
+  }
+
+  #header-container {
+    padding: 2.5rem 15rem;
+  }
+
+  .nav-items li a {
+    text-decoration: none;
+    color: var(--N600);
+  }
+
+  #header-container a {
+    padding: 1rem 1.2rem;
+  }
+
+  #header-container img {
+    display: none;
+  }
+
+  .nav-items ul {
+    list-style: none;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    column-gap: 0.8rem;
+  }
+
+  .nav-items li {
+    padding: 1rem 1.2rem;
+    font-weight: 500;
+    font-size: 1.5rem;
+    line-height: 2rem;
+  }
+
+  .hero-section {
+    background-image: url(../images/desktop-bg.svg);
+    background-size: 100% 100%;
+    height: 100vh;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .hero-section h1 {
+    width: 100%;
+    font-size: 4.8rem;
+    line-height: 6rem;
+  }
+
+  .hero-section p {
+    width: 80%;
+    font-size: 2rem;
+    line-height: 2.8rem;
+  }
+
+  .hero-container {
+    width: 50%;
+  }
+
+  /* Works Section */
+  .works {
+    grid-template-columns: 1fr 3.2fr 1fr;
+  }
+
+  .card-works {
+    width: 100%;
+    height: auto;
+    display: flex;
+    justify-content: space-between;
+    column-gap: 2.2rem;
+    padding: 2.4rem;
+  }
+
+  .card-works:nth-child(2),
```

Comment:
- [ ]  Kindly make sure your cards are responsive and have enough width on the screen of `768px---1024px`. 
As you can see how they look like on your design :point_down: 
![Screenshot from 2022-09-29 18-27-27](https://user-images.githubusercontent.com/97100091/193088086-ef44fcd4-9c26-47d8-8a44-357a07ba937a.png)